src/main/java/Duke.java

```diff
@@ -1,10 +1,17 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Response.greet();
+
+        String input;
+        do {
+            Scanner scan = new Scanner(System.in);
+            input = scan.nextLine();
+
+            Response.parseInput(input);
+        } while (!input.equals("bye"));
```

Comment:
Can squeeze the obtaining of input into a single function to make the code look cleaner.